client/environment_drift_detection.go

```diff
@@ -0,0 +1,31 @@
+package client
+
+func (self *ApiClient) EnvironmentDriftDetection(environmentId string) (EnvironmentSchedulingExpression, error) {
+	var result EnvironmentSchedulingExpression
+
+	err := self.http.Get("/scheduling/drift-detection/environments/"+environmentId, nil, &result)
+	if err != nil {
+		return result, err
+	}
+	return result, nil
+}
+
+func (self *ApiClient) EnvironmentUpdateDriftDetection(environmentId string, payload EnvironmentSchedulingExpression) (EnvironmentSchedulingExpression, error) {
+	var result EnvironmentSchedulingExpression
+
+	err := self.http.Put("/scheduling/drift-detection/environments/"+environmentId, payload, &result)
+	if err != nil {
+		return EnvironmentSchedulingExpression{}, err
+	}
+
+	return result, nil
+}
+
+func (self *ApiClient) EnvironmentStopDriftDetection(environmentId string) error {
+	err := self.http.Put("/scheduling/drift-detection/environments/"+environmentId, EnvironmentSchedulingExpression{Enabled: false, Cron: ""}, &EnvironmentScheduling{})
```

Comment:
We've discussed this briefly at sync today, @alonnoga 's changing the API to be PATCH in [this PR](https://github.com/env0/env0/pull/6758)

Please use `Patch` here

Reply:
7b8a0530(convert to Patch)